.github/workflows/ci.yml

```diff
@@ -18,22 +18,44 @@ jobs:
         override: true
 
     - name: check
-      uses: actions-rs/cargo@v1
-      with:
-        command: check
-        args: --all --benches --bins --examples --tests
+      run: |
+        cargo check --all --benches --bins --examples --tests
+        cargo check --features unstable --all --benches --bins --examples --tests
 
-    - name: check unstable
-      uses: actions-rs/cargo@v1
-      with:
-        command:  check
-        args: --features unstable --all --benches --bins --examples --tests
+    - name: test
+      run: cargo test --all --features unstable
+
+  cross:
+    name: Cross compile
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target:
+          - i686-unknown-linux-gnu
+          - powerpc-unknown-linux-gnu
+          - powerpc64-unknown-linux-gnu
+          - mips-unknown-linux-gnu
+          - arm-linux-androideabi
+
+    steps:
+    - uses: actions/checkout@master
 
-    - name: tests
-      uses: actions-rs/cargo@v1
+    - name: Install nightly
+      uses: actions-rs/toolchain@v1
       with:
-        command: test
-        args: --all --doc --features unstable
```

Comment:
Note that if `cargo test` contains `--doc` arguments, only doc test is executed (fixed in this pr).